rand_distr/src/normal.rs

```diff
@@ -188,6 +188,14 @@ where F: Float, StandardNormal: Distribution<F>
 {
     /// Construct a new `LogNormal` distribution with the given mean
     /// and standard deviation of the logarithm of the distribution.
+    ///
+    /// Note that the parameters are not the resulting mean and standard 
+    /// deviation of the actual samples returned by the LogNormal 
+    /// distribution. Instead, they are the mean and standard deviation 
+    /// of the *logarithm* of those returned values.
+    ///
+    /// Equivalently, the parameters mean and std_dev are the μ(mu) 
+    /// and σ(sigma) parameters of the LogNormal distribution.
```

Comment:
I see, so it is not super clear that the first "distribution" and the second "distribution" are the same. Maybe the following simple clarification is enough?
```suggestion
    /// Construct a new `LogNormal` distribution with the given mean
    /// and standard deviation of the logarithm of the constructed distribution.
```

Reply:
Well, I think it would be nice to have in there that mean and std_dev are mu and sigma. For someone like me, who's not super good at statistics, I (naively) read the wikipedia article about LogNormal, and then tried to use the rand LogNormal. Wikipedia says the parameters of LogNormal are mu and sigma, but those are not to be found in rand.

It's also a bit confusing that the LogNormal distribution actually has a mean, but the parameter 'mean' of its new-function is not that mean.

So I think some sort of 'note' to try and grab the attention of the reader would be of value.

Could one consider renaming mean and std_dev-variables for LogNormal into 'mu' and 'sigma'?